src/Umbraco.Web.UI.Client/src/less/helveticons.less

```diff
@@ -8,35 +8,51 @@
 	font-style: normal;
 }
 
-[class^="icon-"],
-[class*=" icon-"] {
+span[class^="icon-"],
+span[class*=" icon-"] {
+    display: inline-block;
+    width: 1em;
+    height: 1em;
+    *margin-right: .3em;
+
+    svg {
+        width: 100%;
+        height: 100%;
+        fill: currentColor;
+    }
+
+    &.large{
+        width: 32px; 
+        height: 32px;
+    }
+    &.medium{
+        width: 24px;
+        height: 24px;
+    }
+    &.small{
+        width: 14px;
+        height: 14px;
+    }
+}
```

Comment:
Hi @MMasey 
Can I ask you to describe the intention of this piece?
The reason I'm asking because we are looking into reverting parts of this file, to make it possible to have icons in the same manner as previously. aka. spans and buttons with icon-* class. And by making our icons selectors target everything again, I see that colliding with this. Therefore I would remove this part, but let me know the intention of it, what purpose it serves and then we can figure out how this should existing and whether it should.
Thanks in advance, You can also get in touch via Slack if you like?

Reply:
Hey @nielslyngsoe, we may have to update the `umb-icon` to add an attribute or modifier class so that we can detect when we don't want an icon use use the icon font. Maybe something as simple as adding an `icon--svg` when it uses and svg so we can then do something like this

```
.icon {
    // ... icon font code etc

    .icon--svg { // I'm using .icon rather than & here to add a later of specificity to help overwrite the content property
        &::before {
            content: none;
        }
    }
}
```

Reply:
Yeah, that would be cool. Currently, I'm hacking it by looking for spans with icon, but this approach would be way better.

So I'm already working on reverting these parts. But I never finished or tested my work. So if you like please give this a look, and if you like to change something I hope you have access or can make PR towards this one: https://github.com/umbraco/Umbraco-CMS/pull/8941

Reply:
@MMasey could you help me out describing what this part is used for. In my revert PR(https://github.com/umbraco/Umbraco-CMS/pull/8941) I didn't find anything using this part so I have removed it. But that feels wrong. So I hope you could help me out, thanks :-)

Reply:
the diff has a hard time loading, "this part" is referring to:

![image](https://user-images.githubusercontent.com/304656/94445639-f5cdbf00-01a7-11eb-91b1-889f1fb02de0.png)

Reply:
Hey @nielslyngsoe, it's used to ensure the svg icons have the correct sizing, as I think when it was icon fonts it was using font sizing.

Reply:
Also, did you see the PR I created for your PR at #8958?

Reply:
Hi @MMasey Thanks, I actually already figured this out, I took this part into the umb-icon.less, I hope this makes sense to you. :-)

Reply:
I think @nul800sebastiaan is merging the Revert PR now, but please give it a look to see if you think that can work for now? Thanks

Reply:
@nielslyngsoe I've just had another look over your PR at it seems like it has actually remove the functionality to make the svg icons appear in the icon picker.

The revert has renamed `getLegacyIcons` back to `getIcons` in the `iconhelper.service.js` which makes sense, but the other code changes may actually cause issues for backwards compatibility as they were made to help make the fallback work with the `umb-icon`, specifically the formatting of the . Also the `iconpicker.controller.js` no longer utilises the `getAllIcons` method which is responsible to retrieving the svgs.

I think it needs to be something like this (similar to my other PR).

```
iconHelper.getAllIcons()
    .then(icons => {
        vm.icons = icons;
        vm.loading = false;

        iconHelper.getIcons() // Gets legacy icon fronts and appends them to the icon list, removing any duplicates
            .then(icons => {
                if(icons && icons.length > 0) {
                    let legacyIcons = icons.filter(icon => !vm.icons.find(x => x.name == icon.name));
                    vm.icons = legacyIcons.concat(vm.icons);
                }
            });
        });
```


Also, did the the typo correction for the `icon-addressbook.svg` 😉 

I'm happy to have a call to chat about it if it would be easier that going back and forth via this.